packages/x-date-pickers/src/locales/huHU.ts

```diff
@@ -0,0 +1,81 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+// maps TimeView to its translation
+const timeViews = {
+  hours: 'Óra',
+  minutes: 'Perc',
+  seconds: 'Másodperc',
+};
+
+const huHUPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Előző hónap',
+  nextMonth: 'Következő hónap',
+
+  // View navigation
+  openPreviousView: 'Előző nézet megnyitása',
+  openNextView: 'Következő nézet megnyitása',
+  calendarViewSwitchingButtonAriaLabel: (view) =>
+    view === 'year'
+      ? 'az évválasztó már nyitva, váltson a naptárnézetre'
+      : 'a naptárnézet már nyitva, váltson az évválasztóra',
+
+  // DateRange placeholders
+  start: 'Kezdő dátum',
+  end: 'Záró dátum',
+
+  // Action bar
+  cancelButtonLabel: 'Mégse',
+  clearButtonLabel: 'Törlés',
+  okButtonLabel: 'OK',
+  todayButtonLabel: 'Ma',
+
+  // Toolbar titles
+  datePickerToolbarTitle: 'Dátum kiválasztása',
+  dateTimePickerToolbarTitle: 'Dátum és idő kiválasztása',
+  timePickerToolbarTitle: 'Idő kiválasztása',
+  dateRangePickerToolbarTitle: 'Dátumhatárok kiválasztása',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `${timeViews[view] ?? view} kiválasztása. ${
+      time === null
+        ? 'Nincs kiválasztva idő'
+        : `A kiválasztott idő ${adapter.format(time, 'fullTime')}`
+    }`,
+  hoursClockNumberText: (hours) => `${hours} ${timeViews.hours.toLowerCase()}`,
```

Comment:
This is an interesting approach to making sure the capitalization is correct, although, I'm not sure it's necessary to have it perfectly aligned for screen readers. 🙈